lib/request-fragment.js

```diff
@@ -19,8 +19,8 @@ const requiredHeaders = {
  * @param {Object} request - HTTP request stream
  * @returns {Promise} Response from the fragment server
  */
-module.exports = function requestFragment (fragmentUrl, fragmentAttributes, request) {
-    return new Promise((resolve, reject) => {
+module.exports = (fragmentUrl, fragmentAttributes, request) =>
```

Comment:
I have fixed the other comment from @dazld and pushed. Check if it makes sense to merge it?